server.js

```diff
@@ -93,7 +98,7 @@ app.put("/api/movies/:id", (req, res) => {
     }
     return movie;
   });
-  res.status(200).send(req.body);
+  res.status(200).send(movies);
```

Comment:
I think this is smart to match the guided project. I did notice that the delete endpoint doesn't follow this pattern. We may want to think about returning the whole list of movies for delete as well. Then we could just pass setMovies to the Movie component like we're doing elsewhere, and remove the deleteMovie function from App. The instructions in the delete tasks are a bit hard to follow, so this would help that as well.